### DISCUSSION TITLE
Issue with [@types/uswds__uswds]

### DISCUSSION THREAD

**Discussion by hammad-awan** [DefinitelyTyped/DefinitelyTyped#70284] [Issues with a @types package] [answered]
If I do the following in an angular component file:

import { comboBox } from 'uswds__uswds';

I get the following error in the build:

./src/app/shared/components/some-component]ts:7:0-40 - Error: Module not found: Error: Can't resolve 'uswds__uswds' in 'C:\...\src\app\shared\components\some-component'

Even though in VS 2022 if I right click on comboBox in the import statement and "Go to Definition" it takes me to the appropriate index.d.ts file.

[a comment by an automated account is not shown]

**Reply by hammad-awan** [reply]
Doing the following:

import { comboBox } from '@uswds/uswds';

Resolved the angular build issue. I don't know why but it works.

However, the comboBox object has none of the appropriate methods on the object.

I have inclused the appropriate uswds.min.js files as part of the angular.json "scripts" setting.

**Comment by aduth** [comment, marked as answer]
Hello!

What you probably want is:

```ts
import { comboBox } from '@uswds/uswds/js';
// Or:
// import comboBox from "@uswds/uswds/js/usa-combo-box";
```

The double-underscore (`__`) is a DefinitelyTyped convention to represent scoped packages, and `@types/uswds__uswds` represents the types for the `@uswds/uswds` package. But importing `uswds__uswds` is invalid.

`@uswds/uswds` doesn't work as an entrypoint, which is a [known limitation of USWDS](https://github.com/uswds/uswds/issues/4515). Technically this is an inaccuracy in the types, but some bundlers support package aliasing where a developer may choose to remap `@uswds/uswds` to `@uswds/uswds/js';`.
